src/mscorlib/shared/System/IO/PinnedBufferMemoryStream.cs

```diff
@@ -30,18 +30,12 @@ internal PinnedBufferMemoryStream(byte[] array)
             Debug.Assert(array != null, "Array can't be null");
 
             int len = array.Length;
```

Comment:
Nit: Move this line right in front of the `Initialize` call?

Reply:
Moved